fem/restriction.hpp

```diff
@@ -103,12 +118,18 @@ class L2ElementRestriction : public Operator
    L2ElementRestriction(const FiniteElementSpace&);
    void Mult(const Vector &x, Vector &y) const;
    void MultTranspose(const Vector &x, Vector &y) const;
+   void AddMultTranspose(const Vector &x, Vector &y) const;
    /** Fill the I array of SparseMatrix corresponding to the sparsity pattern
        given by this ElementRestriction. */
    void FillI(SparseMatrix &mat) const;
    /** Fill the J and Data arrays of SparseMatrix corresponding to the sparsity
        pattern given by this L2FaceRestriction, and the values of ea_data. */
    void FillJAndData(const Vector &ea_data, SparseMatrix &mat) const;
+   /// @private Not part of the public interface (device kernel limitation).
+   ///
+   /// Performs either MultTranpose or AddMultTranposes depending on the boolean
```

Comment:
typo: `MultTranspose`, `AddMultTranspose`

Reply:
Fixed in 6b38f5c46c96a097d28d374cf5e0fb2ab002e4d5